src/main/java/com/example/demo/OrderComponent/Domain/Entities/Order.java

```diff
@@ -0,0 +1,42 @@
+package com.example.demo.OrderComponent.Domain.Entities;
+
+import com.example.demo.OrderComponent.Domain.Entities.Enums.OrderStatus;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Getter
+@Setter
+@Entity
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID id = UUID.randomUUID();
```

Comment:
Unnecessary to add a random UUID, database does not read this field anyways, it overwrites it with it's own entry.